resources/datadog-integrations/project.rb.erb

```diff
@@ -42,13 +57,34 @@ package :rpm do
   end
 end
 
+# Windows .msi specific flags
+package :msi do
+  # For a consistent package management, please NEVER change this code
+  upgrade_code '<%= guid %>'
+  wix_candle_extension 'WixUtilExtension'
+  wix_light_extension 'WixUtilExtension'
+  if ENV['SIGN_WINDOWS']
+    signing_identity "ECCDAE36FDCB654D2CBAB3E8975AA55469F96E4C", machine_store: true, algorithm: "SHA256"
+  end
+  parameters({
+    'InstallDir' => install_dir,
+    'InstallFiles' => "#{Omnibus::Config.source_dir()}/datadog-agent/dd-agent/packaging/datadog-agent/win32/install_files",
+    'DistFiles' => "#{Omnibus::Config.source_dir()}/datadog-agent/dd-agent/dist",
+    'PackageName' => "DataDog <%= name %> Integration",
+    'IntegrationName' => "<%= name %>",
+    'Description' => " <%= description %>"
+  })
+end
 dependency 'preparation'  # creates required build directories
 dependency 'dd-check-<%= name %>-software'
 
 # Checks should require the datadog agent
 runtime_dependency 'datadog-agent'
 
-extends_packages('datadog-agent', 'datadog')
+if ohai['platform'] == "windows"
```

Comment:
I think it would be better to use one of the following (I know you're not a rubyist, I do this sometimes too in languages I am more unfamiliar with):

```ruby
unless windows?
  #...
end

if !windows?
  #...
end

if ohai['platform'] != 'windows'
  #...
end

unless ohai['platform'] == 'windows'
  #...
end
```